src/decorations/StateDecorations.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  window,
+  OverviewRulerLane,
+  DecorationRangeBehavior,
+  ExtensionContext,
+  TextEditorDecorationType,
+  DecorationRenderOptions,
+} from 'vscode';
+import passingIcon from 'vscode-codicons/src/icons/check.svg';
+import failingIcon from 'vscode-codicons/src/icons/chrome-close.svg';
+import skipIcon from 'vscode-codicons/src/icons/debug-step-over.svg';
+import unknownIcon from 'vscode-codicons/src/icons/question.svg';
+import prepareIcon from './prepareIcon';
+
+export class StateDecorations {
```

Comment:
`prepareIcon` has distinct imports and scoped constant. are you sure it should be moved to `./src/helpers.ts`?

Reply:
> I know our file naming convention is not consistent but based on a few best practices such as here and here, it is probably best to use lowercase, personally I prefer '-' for the separator, but I am open for alternative argument...

Looks like you are still using camelCase for the file names instead of `inline-error.ts`, `test-status.ts` as the best practice links above suggested? 

> prepareIcon has distinct imports and scoped constant. are you sure it should be moved to ./src/helpers.ts?

Not sure I get your point... this to me is a simple utility function that happens to be used in decorations, not the other way around...

Reply:
I'll do it, but I think file names refactor should be done in separate PR. The only right way to enforce **[conventions :wink:](https://github.com/airbnb/javascript#naming--filename-matches-export)** is automated testing. For future reference, this might help: https://www.npmjs.com/package/eslint-plugin-filenames

About helpers, this might be my preference, but I don't like files over 200 lines long.

Reply:
> The only right way to enforce conventions 😉 is automated testing. For future reference, this might help: https://www.npmjs.com/package/eslint-plugin-filenames

good point!